tests/restart_on_config_change_test.yml

```diff
@@ -2,13 +2,13 @@
 - hosts: all
   tasks:
     - name: get docker process PID
-      command: "pgrep -f '/usr/bin/dockerd --raw-logs'"
+      command: "pgrep -f '/usr/bin/docker daemon --raw-logs'"
       register: r_docker_pid
 
 - hosts: all
   vars:
     # same version as already installed
-    docker_version: 1.12.1
+    docker_version: 1.11.2
```

Comment:
i set it to something we are actually using. as you'll notice above there are some differences.